scripts/zones/Uleguerand_Range/npcs/Buffalostalker_Dodzbraz.lua

```diff
@@ -1,22 +1,30 @@
 -----------------------------------
 -- Area: Uleguerand Range
---  NPC: Buffalostalker Dodzbraz
+-- NPC: Buffalostalker Dodzbraz
 -- Type: Quest NPC
 -- !pos -380.171 -24.89 -180.797 5
+-- !additem 1667 (Cluster Core)
 -----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHas(trade, { { 1667, 2 } }) then -- Cluster Core x2
```

Comment:
Please add these items to items.lua so we don't have magic numbers floating around

Reply:
Of course. I wasn't aware they weren't present in that file. I'll add the cores and the salt in there.

Reply:
```
if npcUtil.tradeHas(trade, { { xi.items.CLUSTER_CORE,  2 } }) then
```

Reply:
added to commit